sentry-android-core/src/main/java/io/sentry/android/core/SentryUserFeedbackDialog.java

```diff
@@ -0,0 +1,225 @@
+package io.sentry.android.core;
+
+import android.app.AlertDialog;
+import android.content.Context;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.ImageView;
+import android.widget.TextView;
+import android.widget.Toast;
+import io.sentry.IScopes;
+import io.sentry.Sentry;
+import io.sentry.SentryFeedbackOptions;
+import io.sentry.SentryLevel;
+import io.sentry.SentryOptions;
+import io.sentry.protocol.Feedback;
+import io.sentry.protocol.SentryId;
+import io.sentry.protocol.User;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public final class SentryUserFeedbackDialog extends AlertDialog {
+
+  private boolean isCancelable = false;
+  private @Nullable SentryId currentReplayId;
+  private @Nullable OnDismissListener delegate;
+
+  public SentryUserFeedbackDialog(final @NotNull Context context) {
+    super(context);
+  }
+
+  public SentryUserFeedbackDialog(
```

Comment:
Can we have a ctor or a Dialog builder which takes `SentryFeedbackOptions` as input? IMHO I would even do that in favor of Manifest options, as it allows you to e.g. use localized labels